astroquery/heasarc/__init__.py

```diff
@@ -9,6 +9,9 @@
 The initial version of this was coded in a sprint at the
 "Python in astronomy" workshop in April 2015 by Jean-Christophe Leyder,
 Abigail Stevens, Antonio Martin-Carrillo and Christoph Deil.
+
+Updates to use the XAMIN service was added by Abdu Zoghbi
```

Comment:
All of this and the above author 3 lines can be cleaned-up, too. Git carries the history, and we'll also work on a team website that will list whoever is actively maintain the module.